### ISSUE TITLE
Add missing jump link

### ISSUE THREAD

**Issue by terotik** [kausaltech/kausal-watch-ui#26]
Add "Jump to main content" link to bypass header and move focus to main content

WCAG 2.1  2.4.1 A
